File: classify-server/src/main/java/io/ermdev/alice/dto/ScheduleDto.java
package io.ermdev.alice.dto;

import io.ermdev.alice.entity.Class;
import mapfierj.MapTo;

import java.sql.Time;

public class ScheduleDto {

    private Long id;
    private String day;
    private Time startTime;
    private Time endTime;
    private Double hours;
    private String room;

    @MapTo(ClassDto.class)
    private Class _class ;

    public ScheduleDto(){}
}

File: src/main/kotlin/io/ermdev/classify/dto/ScheduleDto.kt
package io.ermdev.classify.dto

import io.ermdev.classify.hateoas.ResourceSupport
import java.sql.Time

class ScheduleDto(
        var id: Long = 0,

        var day: String = "",

        var room: String = "",

        var start: Time = Time(System.currentTimeMillis()),

        var end: Time = Time(System.currentTimeMillis())) : ResourceSupport()